xbmc/video/VideoDatabase.h

```diff
@@ -401,6 +401,12 @@ const struct SDbTableOffsets DbMusicVideoOffsets[] =
   { VIDEODB_TYPE_INT, my_offsetof(CVideoInfoTag,m_iIdUniqueID)}
 };
 
+typedef enum
```

Comment:
I would suggest an `enum class`. In general we want to get rid of classic enum's and typedef's if possible, because enum classes have some benefits, for example you can forward declare them.